python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -2784,6 +2784,48 @@ def convert_arange(node, **kwargs):
     return nodes
 
 
+@mx_op.register("reverse")
+def convert_reverse(node, **kwargs):
+    """Map MXNet's reverse operator attributes to ONNX
+    """
+    from onnx.helper import make_node
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    axis = int(attrs.get('axis', 0))
+
+    # Transpose takes perm as a parameter, so we must 'pad' the input to a known dim (10 here)
+    perm = [i for i in range(10)]
```

Comment:
what happens if 10 is not enough?

Reply:
I think in mxnet 10-d is the largest you can get

Reply:
I just checked: it seems we can create > 10-d tensors, but I think many ops do not support >= 10d tensors and in general there is no use case for high-dimensional tensors

Reply:
Got it, thanks for the explanation.